tests/msbuild/Xamarin.MacDev.Tasks.Tests/TaskTests/CreateBindingResourceTaskTests.cs

```diff
@@ -139,6 +141,7 @@ void AssertResourceDirectory (string directory, bool symlinks)
 		<ForceLoad></ForceLoad>
 		<Frameworks></Frameworks>
 		<IsCxx></IsCxx>
```

Comment:
What do you think about RequiresSwift instead of IsSwift?

Reply:
@dalexsoto 

That's not a bad idea.

Other ideas: `ContainsSwift`, `UsesSwift`, `ReferencesSwift`, or even `LinkWithSwiftSystemLibraries` to match the msbuild property.

I think they all sound equally good (or bad), so I'm fine with any of these options.

Reply:
I think ‘LinkWithSwiftSystemLibraries’ is the winner for me, it is totally self explanatory / sef documenting on what it’s going on.